packages/synapse-react-client/src/components/doi/DoiFormSchemas.ts

```diff
@@ -0,0 +1,78 @@
+import { RJSFSchema } from '@rjsf/utils'
+
+// The JSON Schema defining the UI form for creating/updating DOIs
+export const doiFormSchema: RJSFSchema = {
+  definitions: {
+    'org.sagebionetworks.repo.model.doi.v2.DoiResourceTypeGeneral': {
+      type: 'string',
+      enum: [
+        'Audiovisual',
+        'Collection',
+        'DataPaper',
+        'Dataset',
+        'Event',
+        'Image',
+        'InteractiveResource',
+        'Model',
+        'PhysicalObject',
+        'Service',
+        'Software',
+        'Sound',
+        'Text',
+        'Workflow',
+        'Other',
+      ],
+    },
+  },
+  type: 'object',
+  properties: {
+    titles: {
+      description: 'The title or titles of your work. One per line.',
+      type: 'array',
+      items: {
+        type: 'string',
+      },
+      title: 'Title(s)',
+    },
+    creators: {
+      title: 'Creator(s)',
+      description:
+        'The main creators or authors of your work, in priority order. These may be the names of actual people and/or institutions.',
+      type: 'array',
+      items: {
+        type: 'string',
+      },
+    },
+    resourceTypeGeneral: {
+      $ref: '#/definitions/org.sagebionetworks.repo.model.doi.v2.DoiResourceTypeGeneral',
+      title: 'General Resource Type',
+      description: 'The general type of content that best matches this work.',
+    },
+    publicationYear: {
+      type: 'integer',
+      title: 'Publication Year',
+      description:
+        'The year your work became or will become public. May be at most one year in the future.',
+    },
```

Comment:
The rest of the schema is purpose-built for the form. Using the raw Synapse OpenAPI schema creates a form that is much more complex than the UI should be.